04a-run_ica.py

```diff
@@ -105,8 +105,8 @@ def run_ica(subject, session=None):
 
     kind = config.get_kind()
     print('Running ICA for ' + kind)
-    ica = ICA(method='fastica', random_state=config.random_state,
-              n_components=n_components[kind])
+    ica = ICA(method='picard', random_state=config.random_state,
```

Comment:
I would actually expose this as a parameter in config as not everyone (sadly) will expect picard to be run by default

Reply:
> not everyone (sadly)

hehe. Hey you need to promote this thing more… and it's not even the default in MNE!!

Reply:
Also I'm taking suggestions here: which parameters should picard receive that are not the defaults? for example here I'm passing `fastica_it=5`, not sure if that is really useful… but maybe you have some ideas about what could make sense?